packages/bruno-electron/src/ipc/network/prepare-request.js

```diff
@@ -61,18 +221,30 @@ const setAuthHeaders = (axiosRequest, request, collectionRoot) => {
         };
         break;
       case 'wsse':
-        var ts = new Date().toISOString();
-        var nonce = crypto_js.lib.WordArray.random(16).toString(crypto_js.enc.Hex);
-        var digest = crypto_js.enc.Base64.stringify(
-          crypto_js.enc.Utf8.parse(
-            crypto_js.SHA1(nonce + ts + get(request, 'auth.wsse.password')).toString(crypto_js.enc.Hex)
-          )
-        );
-
-        axiosRequest.headers['X-WSSE'] = `UsernameToken Username="${get(
-          request,
-          'auth.wsse.user'
-        )}", PasswordDigest="${digest}", Created="${ts}", nonce="${nonce}"`;
+        const user = get(request, 'auth.wsse.user', '');
+        const password = get(request, 'auth.wsse.password', '');
+
+        const ts = new Date().toISOString();
+        const nonce = crypto.randomBytes(16).toString('base64');
+
+        // Create the password digest using SHA-256
+        const hash = crypto.createHash('sha256');
```

Comment:
This logic is no longer correct- the password digest must be computed with a SHA1 hash to be a valid WSSE header

Reply:
Hey, I made the changes because the SHA-1 algorithm was more prone to attacks.

Reply:
True as that may be, API servers don't tend to appreciate arbitrary changes to the standards they're expecting :)

I will open a new feature branch and new pull request to swap to SHA1. If it's not SHA1 it's not WSSE